matrix/common/request/request_cache.py

```diff
@@ -64,6 +67,14 @@ def creation_date(self):
             self._creation_date = item[CacheTableField.CREATION_DATE.value]
         return self._creation_date
 
+    @property
+    def timeout(self) -> bool:
+        timeout = date.to_datetime(self.creation_date) > (date.get_datetime_now()) - timedelta(hours=1)
```

Comment:
I always get this wrong, but are you certain this is `>`?

Reply:
Nope, good call it's the other way. Adding a test